front/src/components/login/register.jsx

```diff
@@ -0,0 +1,189 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import { FormErrors } from "./FormErrors";
+import {authenticationServise} from "../helpers/athentication";
+export class Register extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
There was a similar state initialization somewhere, might want to create the same initialState object for the two.